zap/src/main/java/org/parosproxy/paros/core/scanner/MultipartFormParameter.java

```diff
@@ -44,6 +44,16 @@ public MultipartFormParameter(
         this.type = type;
     }
 
+    public MultipartFormParameter(MultipartFormParameter multipartFormParameter) {
```

Comment:
The changes to this class should be reverted, the variant should adjust the position based on the data injected.

Reply:
yeah sure.